runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/environment/DockerEnvironmentFactory.java

```diff
@@ -182,8 +182,7 @@ private List<String> gcsCredentialArgs() {
     // TODO(https://github.com/apache/beam/issues/19061): Allow this to be disabled manually.
     if (Files.exists(Paths.get(localGcloudConfig))) {
       return ImmutableList.of(
-          "--mount",
-          String.format("type=bind,src=%s,dst=%s", localGcloudConfig, dockerGcloudConfig));
+          "-v", String.format("type=bind,src=%s,dst=%s", localGcloudConfig, dockerGcloudConfig));
```

Comment:
Why are we making this change here?

Reply:
With `--mount` the executions present an error 
` java.io.IOException: Received exit code 125 for command 'docker run -d --mount type=bind,src=/home/actions/.config/gcloud,dst=/root/.config/gcloud --network=host --env=DOCKER_MAC_CONTAINER=null apache/beam_go_sdk:dev --id=3-1 --provision_endpoint=localhost:37971'. stderr: docker: Error response from daemon: invalid mount config for type "bind": bind source path does not exist: /home/actions/.config/gcloud.See 'docker run --help'.` https://github.com/benWize/beam/actions/runs/3178016356/jobs/5189779677, we found out the difference between  `--mount` and `-v` is that -v will create the endpoint when the folder or file doesn't exist.